source/includes/_dataset_security.md

```diff
@@ -0,0 +1,241 @@
+# Dataset security
+
+The dataset security is defined through 3 variables:
+
+* a global accessibility policy
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+### Restricted datasets
+
+If the dataset is [set as restricted](#set-the-global-accessibility-policy), then the dataset will only appear in the catalog for users who have a ruleset declared for them, either directly or through a group. Other users won't have any access to the dataset.
+
+As a result, the default security ruleset has no meaning for restricted datasets.
+
+### Unrestricted datasets
+
+All people having access to the domain will be able to see the dataset in the catalog.
+
+If a user has at least ruleset declared for him/her (directly, through a group or both), he/she will be able to see the union of everything each of the rulesets grants access to.
```

Comment:
we should explain the 'union' behavior:
- AND for the query
- fields + fields for the fields